brigade-worker/src/k8s.ts

```diff
@@ -238,8 +238,12 @@ export class JobRunner implements jobs.JobRunner {
   pod: kubernetes.V1Pod;
   cancel: boolean;
   reconnect: boolean;
+  
+  constructor() {}
 
-  constructor(job: jobs.Job, e: BrigadeEvent, project: Project) {
+  public init<T extends jobs.Job>(job: T, e: BrigadeEvent, project: Project) {
+    // init takes a generic so we can run this against mocks as well as against the real
```

Comment:
nit: can the function comment be before the function definition?

Reply:
fixed